str0m/src/media/receiver.rs

```diff
@@ -152,6 +172,21 @@ impl ReceiverSource {
         self.nacks += 1;
     }
 
+    pub fn update_with_feedback(&mut self, fb: &RtcpFb, now: Instant) {
+        use RtcpFb::*;
+        match fb {
+            SenderInfo(v) => {
+                self.set_sender_info(now, *v);
+            }
+            DlrrItem(v) => {
+                let now = (unix2ntp(SystemTime::now()) >> 16) as u32;
+                let rtt = calculate_rtt_ms(now, v.last_rr_delay, v.last_rr_time);
+                self.rtt = rtt;
+            },
+            _ => {}
+        }
+    }
```

Comment:
I prefer if we stick to splitting the RtcpFb enum on the `MLine` level and have `set_sender_info` and a new `set_dlrr_item`.

Reply:
I don't like 'set_dlrr_item' because it's not setting a dlrr item. I would prefer "update_with_dlrr"

Reply:
`update_with_dlrr` is fine. I only noticed the asymmetry between SenderSource and ReceiverSource now. I pushed a fix for it.